src/rust/collections/async_queue.rs

```diff
@@ -101,6 +101,16 @@ impl<T> AsyncQueue<T> {
     pub fn get_mut_values(&mut self) -> IterMut<T> {
         self.queue.iter_mut()
     }
+
+    /// Get refernce to first item.
+    pub fn get_front(&self) -> Option<&T> {
```

Comment:
Where are these used? Or are they intended for future use?

Reply:
yes, these are for future use. There are some number of functions for future use in that data structure but that one in particular, I will use in the next PR.